bin/benchmark_pydy_code_gen.py

```diff
@@ -5,6 +5,8 @@
 
 # external libraries
 from numpy import hstack, ones, pi, linspace, array, zeros, zeros_like
+import matplotlib
+matplotlib.use('Agg')
```

Comment:
But some people may not have the agg backend install on their on machine (not sure about this, maybe agg is always available on all platforms).

You can check the `TRAVIS` env variable to see if true.